src/server.rs

```diff
@@ -167,6 +167,16 @@ async fn version(
     Ok(HttpResponse::Ok().body(version.to_owned() + "\n"))
 }
 
+fn process_filename_for_url(file_name: &str, config: &RwLock<Config>) -> String {
```

Comment:
I didn't understand why we have 2 almost identical process functions. Can you use an enum and also refactor this?

Reply:
Addressed in [84626fe](https://github.com/omerbustun/rustypaste/commit/84626fe45187d61b3e2f2ed82cf63ea04ee89660).